lib/src/status.rs

```diff
@@ -363,6 +372,7 @@ fn human_render_imagestatus(
     mut out: impl Write,
     slot: Slot,
     image: &crate::spec::ImageStatus,
+    host: &crate::spec::HostStatus,
```

Comment:
@cgwalters Would you be able to explain what you meant by this?
This was in relation to passing through BootEntry instead of HostStatus to the render functions.

> we could pass down the metadata ultimately from ostree_deployment_get_unlocked() here.
https://github.com/bootc-dev/bootc/pull/1291#discussion_r2065157284

I've been trying to wrap my head around how to get the unlocked state from only the Slot, ImageStatus and BootEntry, but can't figure it out.